data/human/human missions.txt

```diff
@@ -7453,3 +7453,442 @@ mission "Adelita 5"
+			choice
+				`	"Can't wait."`
+					goto next2
+				`	"Is that a glorified conference room too?"`
+			`	Howard's smile falters slightly. He forces a light chuckle, though the tension is noticeable. "Well, if I'm being honest, the Sunracer office is... also technically reservable," he admits, rubbing his neck and looking almost pained. "But just as nice! And, uh... look, my real office is technically a cubicle, but compared to the kiosk last time we met? It's a big step up." He shrugs, trying to brush it off. "But hey, we don't need fancy spaces to make serious deals. Let's get back on track, yeah?" He clears his throat.`
+			choice
+				`	"Absolutely."`
+					goto next2
+				`	"Howard, a promotion's a big deal. The office doesn't matter--your work speaks for itself."`
+					goto kindy
+				`	"Howard, your cubicle is still pathetic. Now, let's get back on track."`
+			action
+				"syndicate business: mean to Howard" ++
+			`	Howard flinches, his jaw tightening for a moment before he forces a thin smile. "Pathetic or not, it gets the job done," he says quietly, avoiding your gaze as he fidgets with a paperweight on his desk. "Spaceport. Later." His voice is more clipped now, and he doesn't wait for a response before focusing on his work calls, shutting you out entirely.`
+				decline
+			label kindy
+			action
+				"syndicate business: mean to Howard" --
+			`	"Thanks, that means a lot. Now, I've got to get back to work."`
+			label next2
+			`	Howard's attention shifts away from you as he begins taking a series of work calls, his tone shifting seamlessly from charming to authoritative as he negotiates and makes deals. He mouths two words to you: "Spaceport... later..." It's clear that, for now, Howard is finished with you and you'll get no more information.`
+
+
+mission "Syndicate Business 4"
+	name "Escort Howard to <planet>"
+	description "Escort Howard from Megaparsec Inc. to <planet> to learn what favor is needed in exchange for an investment opportunity."
+	priority
+	source "Hephaestus"
+	to offer
+		has "Syndicate Business 3: done"
+	destination "Sunracer"
+	on offer
+		conversation
+			`You meet Howard at the spaceport. He's standing on a nearby landing pad by his Shuttle, dressed sharply but looking a bit hurried. He gives you a quick nod as you approach.`
+			`	"Good to see you're on time," Howard says, glancing at his antique wristwatch. "We've got a lot of ground--or space--to cover." Howard briefly looks at the <ship>. "Your ship ought to go fast enough, I suppose. I'll be tied up in calls for most of the trip, but once we're back at the office, we can talk." He starts toward his Shuttle's ramp, already shifting his attention back to his wristwatch. "Let's get moving."`
```

Comment:
I suppose another question would be how can he have calls while piloting?
You could perhaps make this a Bounder/Arrow, to convey a slightly larger feeling of opulence?

Reply:
My thought was that he'd take calls while piloting much like people take hands-off calls while driving, or how aircraft pilots chat over radio while flying.

For the ship, my plan is for him to get a Bounder after this mission in Part 3 when he really gets a good promotion. As a rough outline for my thoughts, the first mission he was essentially a nobody, this mission he's still junior but starting to get some resources (shuttle and a cubicle office, and he has to rent conference rooms for the "nice" offices), and Part 3 will have the Bounder, a permanent nice office, perhaps an assistant. And then even more opulent in Part 4 (he'll get a Marauder Bounder if the Marauder path stays, maybe a couple escorts, penthouse office, etc).